.github/workflows/weekly_tests.yaml

```diff
@@ -72,20 +75,32 @@ jobs:
           - repo: charm-userdir-ldap
             workflow_file_name: check.yaml
             branch: master
-          - repo: prometheus-hardware-exporter
+          - repo: charmed-openstack-upgrader
             workflow_file_name: check.yaml
             branch: main
-          - repo: prometheus-juju-exporter
-            workflow_file_name: pr.yaml
+          - repo: hardware-observer-operator
+            workflow_file_name: check.yaml
+            branch: master
+          - repo: hardware-observer-operator
+            workflow_file_name: cos_integration.yaml
+            branch: master
+          - repo: layer-beats-base 
+            workflow_file_name: ci.yaml
+            branch: master
```

Comment:
The `ci.yaml` for layer-beats-base only contains lint and unit tests. So I don't think it deserves to be run weekly.